MetaMask/metamask-extension
# feat: update snap label to include "Beta" keyword

## **Description**

Update the snap account label from `Snaps` to `Snaps (Beta)`. 

## **Manual testing steps**

1. Install the [Simple Snap Keyring](https://metamask.github.io/snap-simple-keyring/latest/)
2. Create a new snap account
3. Notice the label says `Snaps (Beta)`

## **Screenshots/Recordings**

_[screenshot]_

<img width="321" alt="Screenshot 2023-10-06 at 00 41 01" src="https://github.com/MetaMask/metamask-extension/assets/17601467/7aa148a7-bd1b-4ac3-aff5-95c2e74f74b2">

## **Related issues**

Fixes https://github.com/MetaMask/accounts-planning/issues/88

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
